### ISSUE TITLE
How to change   "useAppcCLI"

### ISSUE THREAD

**Issue by furthurr** [dbankier/TiShadow#513]
I have problem with  "useAppcCLI" ...
how change `  "useAppcCLI": true`, --- to ---  `"useAppcCLI": false`,

**Comment by dbankier** [comment]
Good point. Need to fix that. For the moment manually just edit `~/.tishadow.json`
